lib/livebook/session/recently_opened.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Livebook.Session.RecentlyOpened do
+  @moduledoc false
+
+  use GenServer
+
+  alias Livebook.Session
+  alias Livebook.Users.User
+  alias Livebook.Storage
+
+  @namespace :recently_opened_sessions
+  @max_size 10
+
+  def start_link(arg) do
+    GenServer.start_link(__MODULE__, arg, name: __MODULE__)
+  end
+
+  def list_sessions(user) do
+    GenServer.call(__MODULE__, {:list_sessions, user})
+  end
+
+  @spec save_session(User.t(), Session.t()) :: :ok
+  def save_session(user, session) do
```

Comment:
Livebook doesn't really have permanent users, we do store some preferences/id in the cookie, but opening Livebook in another browser or private window would clear it. Given that, I think we should keep track of recent sessions regardless the user.

Reply:
If it's not related to the user, then what's the difference with "RUNNING SESSIONS"?

Reply:
The difference is that we will show recent notebooks, even when all sessions are closed (in particular when Livebook is restarted). It's basically a way to reopen closed notebooks that you worked on recently.